public/js/main.js

```diff
@@ -0,0 +1,31 @@
+var displayResults = function(data) {
+  var found = data["query"]["searchinfo"]["totalhits"];
+  if (found > 0) {
+    var array = data["query"]["search"];
+    var mainLink = "https://en.wikipedia.org/wiki/";
+    array.forEach(function(searchResult) {
+      var pgLink = mainLink + searchResult["title"].split(" ").join("_");
+      $("#results").append(
+        "<div class='unit'><a target ='_blank' href='" + pgLink + "'> <div class = 'title'>" +
+        searchResult["title"] +
+        "</div><div class='description'>" +
+        searchResult["snippet"] +
+        "</div></a></div>");
+    });
+  } else {
+    $("#analysis").html("No result found");
+  }
+}
+
+$(document).ready(function() {
+  $("#submitButton").click(function() {
+    $("#analysis").empty();
+    var searchInput = $(".search").val();
```

Comment:
As searchInput is used as an URL parameter, it could be safer to use encodeURIComponent to avoid problem (for example, if you search for "Ben & Jerry's" it probably fails without)